resqpy/model.py

```diff
@@ -2951,6 +2954,20 @@ def md_datums(self):
          datum = resqpy.well.MdDatum(self, md_datum_root=datum_root)
          yield datum
 
+   def iter_crs(self):
+      """Iterable of all CRS objects associated with the model
+      
+      Yields:
+         crs: instance of :class:`resqpy.crs.CRS`
+
+      :meta common:
+      """
+      import resqpy.crs
+
+      uuids = self.uuids(obj_type = 'LocalDepth3dCrs')
```

Comment:
The resqpy Crs class also deals with the very similar RESQML class LocalTime3dCrs (if I remember correctly). I think this iterator should yield all the resqpy Crs objects possible, ie. after the depth ones, continue with the time ones.